### ISSUE TITLE
scripts missing

### ISSUE THREAD

**Issue by riyansha** [vinceasvp/meta-sc#1]
normalize-resample.sh and construct-LS100.py scripts cannot be found

**Comment by moshengmao** [comment]
Yes, I have the same issue. Have you resolved it yet? I also couldn't find the `normalize-resample.sh` and `construct-LS100.py` scripts.

**Comment by sunyulin0421** [comment]
Same question
